### DISCUSSION TITLE
Adonis Drive issues with SSL Certificate?

### DISCUSSION THREAD

**Discussion by tiagocdr** [adonisjs/core#3555] [Help]
So I deployed an AdonisJS 5 server with nginx and added the ssl certificate trough certbot, it automates the renewal process every 3 months; before the renewal we kept adding new features and one of them was hosting images straight from the server, with and without signed urls. 

After we pushed thosed updates we realized the SSL Certificate didn't renew and the clients started failing, I realized certbot makes a request to the server to a given path that clearly doesn't exist, this time, it gets a 404 response and doesn't renew automatically after failing

Has this happened to anyone else? Is it an intented behavior with Drive, I'm just assuming so I'm not completely sure what is the cause, it could also be something else that I'm not grasping.

I'll post some error messages of the situation to be more detailed.

**Comment by galaczi** [comment]
I haven't worked with nginx for a long time but if I remember correctly, you need to have a separate server block for cerbot. Also reload the nginx config after each renewal.
